sync/class.jetpack-sync-module-posts.php

```diff
@@ -116,8 +116,16 @@ function filter_post_content_and_add_links( $post_object ) {
 		if ( 0 < strlen( $post->post_password ) ) {
 			$post->post_password = 'auto-' . wp_generate_password( 10, false );
 		}
-		/** This filter is already documented in core. wp-includes/post-template.php */
 		if ( Jetpack_Sync_Settings::get_setting( 'render_filtered_content' ) ) {
+
+			remove_filter( 'the_content', array( Jetpack_Likes::init(), 'post_likes' ), 30 );
+			remove_filter( 'the_excerpt', array( Jetpack_Likes::init(), 'post_likes' ), 30 );
+			remove_filter( 'the_content', array( Jetpack_RelatedPosts::init(), 'filter_add_target_to_dom' ), 40 );
+			remove_filter( 'the_content', array( Jetpack_RelatedPosts::init(), 'test_for_shortcode' ), 0 );
```

Comment:
Does it make sense to group these by `the_content` and the `excerpt`? At first glance, I thought we were removing three filters from the content and the excerpt. But, after reading more, it looks like we're actually removing 2 from `the_excerpt` and 4 from `the_content`.